content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+| How to reproduce published Galaxy Analysis |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-reproduce/tutorial.html){:target="_blank"} |   |   |
+|Introduction to the GTN |[<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/slides/introduction.html#1
+){:target="_blank"} |   | [<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=lDqWxzWNk1k){:target="_blank"} |   |
+| How to contribute using Gitpod |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/gitpod/tutorial.html){:target="_blank"}  | [<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/_-wnsPttBCo){:target="_blank"}  |   |
+|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |
+
+### Useful links
+- [FAQs quickstart](https://training.galaxyproject.org/training-material/faq)
+  
+  Feel like you might have a question that many others have had as well? Check our Frequently Asked Questions quickstart!
+
+- [Public Galaxy Servers](https://galaxyproject.org/use/)
+  
+  List of all Public Galaxy Servers available to use or deploy your own version.
+
+- [Galaxy Training Network](https://training.galaxyproject.org/)
+  
+  Collection of tutorials from a variety of topics() developed and maintained by the worldwide Galaxy community.
+
+- [Galaxy Help](https://help.galaxyproject.org/)
+  
+  Have a bug or an error you haven't been able to solve? Do you have a question that you haven't found in already answered in the FAQs? Reach out to the Galaxy Help Community.
+
+- [Galaxy Communities](https://galaxyproject.org/community/)
+  
+  List and description of already stablished communities.
+
+- [Galaxy Publication Library](https://www.zotero.org/groups/1732893/galaxy)
+  
+  Publication library for the Galaxy Project that use, reference, implement, and extend Galaxy.
+
+- [Events](https://galaxyproject.org/events/)
+  
+  Upcoming and past events with Galaxy-related content.
+
+- [Mailing lists](https://galaxyproject.org/mailing-lists/)
+  
+  Links, email address and archives for Galaxy's Mailing Lists, you can reach out to the community in this format, ask your questions and get to know what's up with Galaxy.
+
+- [Working groups](https://galaxyproject.org/community/wg/)
+  
```

Comment:
I would also add a bit more background information. Why a newbee would be interested by working groups e.g. at which stage of his Galaxy journey.  I thin kit is more when you think about "contributing" to Galaxy so it could be in a "Ready to contribute"  section.